frontend/src/pages/modelRegistrySettings/CreateMRSecureDBSection.tsx

```diff
@@ -58,6 +63,39 @@ export const CreateMRSecureDBSection: React.FC<CreateMRSecureDBSectionProps> = (
     return false;
   };
 
+  const clusterWideCABundle = existingCertConfigMaps.find(
+    (configMap) => configMap.name === ODH_TRUSTED_BUNDLE && configMap.keys.includes(CA_BUNDLE_CRT),
+  );
+
+  const isClusterWideCABundleAvailable = !!clusterWideCABundle;
+
+  const openshiftCAbundle = existingCertConfigMaps.find(
+    (configMap) =>
+      configMap.name === ODH_TRUSTED_BUNDLE && configMap.keys.includes(ODH_CA_BUNDLE_CRT),
+  );
+
+  const isProductCABundleAvailable = !!openshiftCAbundle;
+
+  const getKeysByName = (
+    configMaps: ConfigSecretItem[],
+    secrets: ConfigSecretItem[],
+    targetName: string,
+  ): string[] => {
+    let keys: string[] = [];
+    // Search in ConfigMaps first
+    const configMap = configMaps.find((item) => item.name === targetName);
+    if (configMap) {
+      keys = configMap.keys;
+    }
+
+    // If not found, search in Secrets
+    const secret = secrets.find((item) => item.name === targetName);
+    if (secret) {
+      keys = secret.keys;
+    }
+    return keys;
```

Comment:
Wouldn't this be a problem if we have a configmap and secret with the same name and the user is trying to select the secret? I think we can maybe pass only the list of ConfigSecretItems of the type being selected here.